heft-plugins/heft-typescript-plugin/src/TypeScriptBuilder.ts

```diff
@@ -1297,12 +1297,16 @@ export class TypeScriptBuilder {
   }
 }
 
+type BuilderProgramWithInternals = TTypescript.BuilderProgram & {
+  state?: { changedFilesSet: Set<string> };
+  getState(): { changedFilesSet: Set<string> };
+};
```

Comment:
This should probaly go in https://github.com/microsoft/rushstack/blob/main/heft-plugins/heft-typescript-plugin/src/internalTypings/TypeScriptInternals.ts.